### ISSUE TITLE
Not Working in React Strict Mode

### ISSUE THREAD

**Issue by Vitorbnc** [material-table-core/core#226]
# Guidelines

**Describe the bug**
Icons don't appear and and an error shows up in console. The error is as follows:

>index.js:1 Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Transition which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://reactjs.org/link/strict-mode-find-node
    at div
    at Transition (http://localhost:3000/static/js/vendors~main.chunk.js:95502:30)
    at Grow (http://localhost:3000/static/js/vendors~main.chunk.js:21102:24)
    at div
    at Portal (http://localhost:3000/static/js/vendors~main.chunk.js:29447:24)
    at Popper (http://localhost:3000/static/js/vendors~main.chunk.js:29126:24)
    at Tooltip (http://localhost:3000/static/js/vendors~main.chunk.js:39718:28)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at div
    at MTablePagination (http://localhost:3000/static/js/vendors~main.chunk.js:3055:10)
    at MTablePaginationRef
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at ActionsComponent
    at div
    at Toolbar (http://localhost:3000/static/js/vendors~main.chunk.js:39434:23)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at td
    at TableCell (http://localhost:3000/static/js/vendors~main.chunk.js:36682:28)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at TablePagination (http://localhost:3000/static/js/vendors~main.chunk.js:37226:37)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at tr
    at TableRow (http://localhost:3000/static/js/vendors~main.chunk.js:37643:23)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at tfoot
    at TableFooter (http://localhost:3000/static/js/vendors~main.chunk.js:36945:23)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at table
    at Table (http://localhost:3000/static/js/vendors~main.chunk.js:36301:23)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at div
    at Paper (http://localhost:3000/static/js/vendors~main.chunk.js:28361:23)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at Container (http://localhost:3000/static/js/vendors~main.chunk.js:394:27)
    at ContainerRef
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:91681:20)
    at App (http://localhost:3000/static/js/vendors~main.chunk.js:62505:25)
    at ErrorBoundary (http://localhost:3000/static/js/vendors~main.chunk.js:55703:35)
    at DragDropContext (http://localhost:3000/static/js/vendors~main.chunk.js:62637:19)
    at MaterialTable (http://localhost:3000/static/js/vendors~main.chunk.js:6488:37)
    at http://localhost:3000/static/js/vendors~main.chunk.js:6340:16
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at Main
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:94154:29)
    at PrivateRoute (http://localhost:3000/static/js/main.chunk.js:5512:14)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:94356:29)
    at main
    at div
    at div
    at MiniDrawer (http://localhost:3000/static/js/main.chunk.js:2800:3)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:94356:29)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:93789:30)
    at HashRouter (http://localhost:3000/static/js/vendors~main.chunk.js:93454:35)
    at Routes
    at App
    at CssBaseline (http://localhost:3000/static/js/vendors~main.chunk.js:15878:31)
    at WithStyles (http://localhost:3000/static/js/vendors~main.chunk.js:53036:31)
    at ThemeProvider (http://localhost:3000/static/js/vendors~main.chunk.js:51738:24)
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:91681:20)
console.<computed> @ index.js:1
printWarning @ react-dom.development.js:67
error @ react-dom.development.js:43
findHostInstanceWithWarning @ react-dom.development.js:25406
findDOMNode @ react-dom.development.js:26067
performEnter @ Transition.js:245
updateStatus @ Transition.js:228
componentDidMount @ Transition.js:172
commitLifeCycles @ react-dom.development.js:20663
commitLayoutEffects @ react-dom.development.js:23426
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
commitRootImpl @ react-dom.development.js:23151
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
commitRoot @ react-dom.development.js:22990
performSyncWorkOnRoot @ react-dom.development.js:22329
(anonymous) @ react-dom.development.js:11327
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushSyncCallbackQueueImpl @ react-dom.development.js:11322
flushSyncCallbackQueue @ react-dom.development.js:11309
scheduleUpdateOnFiber @ react-dom.development.js:21893
dispatchAction @ react-dom.development.js:16139
(anonymous) @ useControlled.js:38
handleOpen @ Tooltip.js:273
(anonymous) @ Tooltip.js:306
setTimeout (async)
(anonymous) @ Tooltip.js:305
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js:4070
executeDispatch @ react-dom.development.js:8243
processDispatchQueueItemsInOrder @ react-dom.development.js:8275
processDispatchQueue @ react-dom.development.js:8288
dispatchEventsForPlugins @ react-dom.development.js:8299
(anonymous) @ react-dom.development.js:8508
batchedEventUpdates$1 @ react-dom.development.js:22396
batchedEventUpdates @ react-dom.development.js:3745
dispatchEventForPluginEventSystem @ react-dom.development.js:8507
attemptToDispatchEvent @ react-dom.development.js:6005
dispatchEvent @ react-dom.development.js:5924
unstable_runWithPriority @ scheduler.development.js:468
dispatchUserBlockingUpdate @ react-dom.development.js:5894

**To Reproduce**

1. Run the styling example from _material-table_ docs.

**Screenshots**
![image](https://user-images.githubusercontent.com/7032355/121689221-bd9eb000-ca9a-11eb-95f9-1b2c7f042337.png)

![image](https://user-images.githubusercontent.com/7032355/121689465-022a4b80-ca9b-11eb-8dd8-225b27a6ff79.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome
- Version: 90

**Comment by matthewoestreich** [comment]
Do we know if this is from a dependency or not? I've seen that before and want to say it may be an issue with a dep.

I can check here in a few minutes.

**Comment by Vitorbnc** [comment]
I fixed the icons problem. It's not actually a problem and it's explained [in the material-table docs](https://material-table.com/#/docs/install).

The `findDOMNode` error also occurs in the original [material-table repo](https://github.com/mbrn/material-table/issues/2345). It's probably related to material-ui itself. 
There's an issue [here](https://github.com/mui-org/material-ui/issues/13394), and they say it's possible to alter the imports to fix it for now, but it should be fixed in material-ui v5. I don't know what is causing it in this library though, I've used `createMuiTheme` before and there were no warnings.
